lib/chess/core/pieces/pawn.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'base_piece'
+
+module Chess
+  module Core
+    module Pieces
+      class Pawn < BasePiece
+        SYMBOL = :P
+        TEXT = "\u265f".freeze
+
+        def self.create_and_occupy(team, current_cell)
+          self_pawn = super(SYMBOL, team, current_cell)
+          @first_move = true
```

Comment:
what do you need this for?

Reply:
No, it was deprecated by first_move? method